src/scanner.rs

```diff
@@ -1426,3 +1677,105 @@ fn check_fn_for_effects(scan_results: &ScanResults, fn_: &CanonicalPath) -> bool
 
     false
 }
+
+fn format(
+    db: &RootDatabase,
+    kind: SyntaxKind,
+    file_id: FileId,
+    expanded: SyntaxNode,
+) -> String {
+    let expansion = insert_ws_into(expanded).to_string();
+
+    _format(db, kind, file_id, &expansion).unwrap_or(expansion)
+}
+
+fn _format(
```

Comment:
If this is unused I would just remove for now - what is it going to be used for hypothetically?